lisa/tools/xfstests.py

```diff
@@ -114,7 +114,7 @@ class Xfstests(Tool):
 
     @property
     def command(self) -> str:
-        return "xfstests"
+        return str(self._code_path)
```

Comment:
It looks is the folder "/xfstest-dev" does it work as the command here?

Reply:
command is not used in xfstests at all, change into 'f"ls -lt {self._code_path}"'?

Reply:
ok, so it looks like used for exists check only. If so, the code is ok. But you can add some comments to explain the weird logic.

Reply:
updated